Content.Server/PDA/PDAComponent.cs

```diff
@@ -59,9 +56,22 @@ public class PDAComponent : SharedPDAComponent, IInteractUsing, IActivate, IUse,
 
         [ViewVariables] private BoundUserInterface? UserInterface => Owner.GetUIOrNull(PDAUiKey.Key);
 
-        [DataField("insertIdSound")] private SoundSpecifier _insertIdSound = new SoundPathSpecifier("/Audio/Weapons/Guns/MagIn/batrifle_magin.ogg");
-        [DataField("toggleFlashlightSound")] private SoundSpecifier _toggleFlashlightSound = new SoundPathSpecifier("/Audio/Items/flashlight_toggle.ogg");
-        [DataField("ejectIdSound")] private SoundSpecifier _ejectIdSound = new SoundPathSpecifier("/Audio/Machines/id_swipe.ogg");
+        [ViewVariables]
+        [DataField("idCard")]
+        private string? _startingIdCard = "AssistantIDCard";
+
+        [ViewVariables]
+        [DataField("pen")]
+        private string? _startingPen = "Pen";
```

Comment:
Probably a bad idea to move stuff around when an existing PR is open (PDA ECS).

Reply:
True, true.